pkg/ccl/importccl/import_stmt.go

```diff
@@ -1940,18 +1891,6 @@ func (r *importResumer) Resume(ctx context.Context, execCtx interface{}) error {
 	details := r.job.Details().(jobspb.ImportDetails)
 	files := details.URIs
 	format := details.Format
-	ptsID := details.ProtectedTimestampRecord
```

Comment:
Do we need to keep the cleanup around to free protections reserved by jobs on older binaries?

Reply:
good catch, readded the release logic. The helper we use to release has a nil check for `details.ProtectedTimestampRecord`, so for a 21.2 node this will just be a noop.